CONTRIBUTING.md

```diff
@@ -60,8 +60,6 @@ A good javadoc example taken from [`AbstractCharSequenceAssert.containsSequence`
  */
 ```
 
-Not all AssertJ javadoc meets this standard but this is something we are working on.
```

Comment:
Now that all or almost all javadoc meets this standard I think this qualification is not necessary anymore. :)